app/javascript/packs/inputServiceWorker.ts

```diff
@@ -0,0 +1,22 @@
+// Specifically import service worker provided by the Papyros-package
+import { InputWorker } from "@dodona/papyros/dist/workers/input/InputWorker";
```

Comment:
Since `InputWorker` is exported in [papyros' `Libary.ts`](https://github.com/dodona-edu/papyros/blob/main/src/Library.ts), you should be able to change this to:
```suggestion
import { InputWorker } from "@dodona/papyros";
```

Reply:
Ah, sad. Maybe it is a good idea to mention the reason for this type of import in a comment then?

Reply:
The comment indeed does not mention this specific issue, only that I specifically import the worker. I'll update it to be more clear.